src/neptune/metadata_containers/metadata_container.py

```diff
@@ -315,7 +314,7 @@ def _write_initial_attributes(self):
 
     def __exit__(self, exc_type, exc_val, exc_tb):
         if exc_tb is not None:
-            traceback.print_exception(exc_type, exc_val, exc_tb)
+            uncaught_exception_handler.trigger(exc_type, exc_val, exc_tb)
```

Comment:
Will `traceback.print_exception` still be covered?

Reply:
I'm not sure it needs to be tbh - we want to have the same output whether we're using context manager or not, right?

Reply:
In case of an error in a context, nothing is being printed to `stderr`.
Expected behaviour?
![image](https://github.com/neptune-ai/neptune-client/assets/41324509/fddbe216-974c-4647-bf6d-6ba9196a5469)

Reply:
So let's trigger the uncaught exception handler + leave print_exception as it was :wink: